app/assets/stylesheets/components/_restaurant_cards.scss

```diff
@@ -8,10 +8,23 @@
 .card {
   border: 1px solid black;
   width: 30px;
-  height: 30px;
```

Comment:
can you make a specific class for this? like card_restaurant or something? Otherwise you will touch all the cards also cards with class cards_dishes